modules/sdk-core/src/bitgo/wallet/iWallet.ts

```diff
@@ -57,6 +57,7 @@ export interface PrebuildTransactionOptions {
   recipients?: {
     address: string;
     amount: string | number;
+    walletId?: string;
```

Comment:
`address` is currently a required field in `recipients`

I think we should make it optional and add validation to ensure that either `address` or `walletId` is provided (XOR)

Reply:
it also depends on what we do in the UI and how we parse it in WP.

If we support only one box field for user to input either address or wallet id, it will be the case that the `address` is used for both address and wallet id, we dont necessary need to add the `walletId` field, but to have address accept both? it would maybe cause some confusion.

Reply:
We'll have a specific input for `walletId` in this case. I'd prefer to have `walletId` as a separate param, so we don't have to worry about address validations breaking if a wallet id is passed.

<img width="589" alt="image" src="https://user-images.githubusercontent.com/4226651/198393154-a85b2da6-f618-4b3b-b02c-448221646d9a.png">